swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/VPNConfigurationManager.swift

```diff
@@ -96,42 +96,46 @@ public class VPNConfigurationManager {
 
     let ipcClient = IPCClient(session: session)
 
+    var userDict: [String: Any?] = [:]
     var migrated = false
 
-    if let apiURL = legacyConfiguration["apiURL"] {
-      try await ipcClient.setApiURL(apiURL)
+    if let actorName = legacyConfiguration["actorName"] {
+      UserDefaults.standard.set(actorName, forKey: "actorName")
       migrated = true
     }
 
-    if let authURL = legacyConfiguration["authBaseURL"] {
-      try await ipcClient.setAuthURL(authURL)
+    if let apiURL = legacyConfiguration["apiURL"] {
+      userDict[Configuration.Keys.apiURL] = apiURL
       migrated = true
     }
 
-    if let actorName = legacyConfiguration["actorName"] {
-      try await ipcClient.setActorName(actorName)
+    if let authURL = legacyConfiguration["authBaseURL"] {
+      userDict[Configuration.Keys.authURL] = authURL
       migrated = true
     }
 
     if let accountSlug = legacyConfiguration["accountSlug"] {
-      try await ipcClient.setAccountSlug(accountSlug)
+      userDict[Configuration.Keys.accountSlug] = accountSlug
       migrated = true
     }
 
     if let logFilter = legacyConfiguration["logFilter"],
        !logFilter.isEmpty {
-      try await ipcClient.setLogFilter(logFilter)
+      userDict[Configuration.Keys.logFilter] = logFilter
       migrated = true
     }
 
     if let internetResourceEnabled = legacyConfiguration["internetResourceEnabled"],
        ["false", "true"].contains(internetResourceEnabled) {
-      try await ipcClient.setInternetResourceEnabled(internetResourceEnabled == "true")
+      userDict[Configuration.Keys.internetResourceEnabled] = internetResourceEnabled == "true"
       migrated = true
     }
 
     if !migrated { return }
 
+    let configuration = Configuration(userDict: userDict, managedDict: [:])
```

Comment:
This is a bit of a hack now in order to bulk save the configuration, but since it's a migration we'll be removing, not sure it's worth adding another constructor to Configuration.